.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ccpp-physics"]
```

Comment:
@GeorgeGayno-NOAA  You are going to check out the whole  "ccpp-physics" submodule?

Reply:
Do you know a way to checkout just a single file?